src/index.js

```diff
@@ -42,6 +41,10 @@ const patch = snabbdom.init([
 		votes.push(vote)
 	}
 
+	const commit = async () => {
```

Comment:
Not sure a single `await` is really worth the async function. I'd rather use `async`/`await` below, in `onSubmit`.

Reply:
So you are saying to move it to `onSubmit`?

Reply:
yup. just `await archive.commit()` in `onSubmit`.